directory-listing/darkmode.css

```diff
@@ -0,0 +1,28 @@
+body{
+  background-color: #333333;
```

Comment:
We use colors from Google's Material design system already, so try to pick colors from that [list](https://ramlmn.github.io/material-colors.html) only.